src/System.Private.ServiceModel/src/System/ServiceModel/Description/TypeLoader.cs

```diff
@@ -608,8 +608,14 @@ internal void AddBehaviorsFromImplementationType(ServiceEndpoint serviceEndpoint
         {
             foreach (IEndpointBehavior behaviorAttribute in ServiceReflector.GetCustomAttributes(implementationType, typeof(IEndpointBehavior), false))
             {
-                // 
-                throw ExceptionHelper.PlatformNotSupported();
+                if(behaviorAttribute is CallbackBehaviorAttribute)
+                {
+                    serviceEndpoint.Behaviors.Insert(0, behaviorAttribute);
```

Comment:
I have to add this code to prevent PlatformNotSupported exception thrown by the added test, and I go ahead to do similar update to the TypeLoader.cs used by dotnet-svcutil in this same PR.

Reply:
Please do.